setup.cfg

```diff
@@ -62,6 +62,9 @@ install_requires =
     PyJWT~=2.6
     # Capture stdout/stderr
     wurlitzer~=3.0
+    pyarrow # For dremio client
+    pandas # For dremio client
+    certifi # For dremio client
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Do we have a min version of these APIs we're using?

Reply:
Good remark. I wasn't sure it's a good time investment but bisecting versions was a fun exercise! I also learned about that root certificate thing.